plugins/survey-resources/src/components/OptionsQuestionDataEditor.svelte

```diff
@@ -0,0 +1,135 @@
+<!--
+// Copyright © 2023 Hardcore Engineering Inc.
+//
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+
+<script type="module" lang="ts">
+  import { Button, EditBox, RadioButton, IconDelete, IconAdd, CheckBox } from '@hcengineering/ui'
+  import { type Checkboxes, QuestionDataEditorComponentProps, type RadioButtons } from '@hcengineering/survey'
+  import survey from '../plugin'
+
+  type Data = Checkboxes | RadioButtons
+
+  /**
+   * Declared $$Props help TypeScript ensure that your component is a valid {@link QuestionDataEditorComponent}
+   * @see https://raqueebuddinaziz.com/blog/svelte-type-events-slots-and-props/#restprops-props
+   */
+  interface $$Props extends QuestionDataEditorComponentProps<Data> {}
+
+  export let object: Data
+  export let editable = true
+  export let submit: (data: Data) => Promise<void>
```

Comment:
Submit is passed as a callback prop here (instead of event), because it's a part of common interface for all existing and future question data editors, and some of them might need to call this callback inside to wait for its result